answerbot_sample/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        applicationId rootProject.ext.applicationId + ".answerbot"
+        applicationId rootProject.ext.applicationId + ".answerbot_sample"
```

Comment:
Just curious why we updated the application id for this.

Reply:
just to have the same name as the package, and that way all app have the same applicationId format